### PR TITLE
Gocdserver

### PR DESCRIPTION
**Pull request by awielern**
Please your review

### CODE REVIEW

File: acceptance.gocd.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    stages:
+    - Deploy:
+        fetch_materials: false
+        jobs:
+          docker:
+            resources:
+              - docker
+            timeout: 0
+            tasks:
+            - fetch:
```
Comment (vcarmen) [inline acceptance.gocd.yaml]: you don't need to fetch the materials

File: commit.gocd.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              - ubuntu
+            timeout: 0
+            tasks:
+            - exec:
+                arguments:
+                - -c
+                - pip3 install -r ConvertVideo/requirements.txt
+                command: bash
+                run_if: passed
+            - exec:
```
Comment (vcarmen) [inline commit.gocd.yaml:38]: Include execution of unit tests

**Review by vcarmen — CHANGES_REQUESTED** [review]
Remove not needed files from the PR

**Comment by awielern** [comment]
Correcting the request

**Pull request closed without merging**